integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/LoadBalanceEndpointTestCase.java

```diff
@@ -99,122 +68,49 @@ public void close() throws Exception {
         if (axis2Server3.isStarted()) {
             axis2Server3.stop();
         }
-        resourceAdminServiceClient.deleteResource("/_system/config/test_ep_config/");
+
         axis2Server1 = null;
         axis2Server2 = null;
         axis2Server3 = null;
-        resourceAdminServiceClient = null;
-        endPointAdminClient = null;
         lbClient = null;
-        super.cleanup();
-    }
-
-    @SetEnvironment(executionEnvironments = { ExecutionEnvironment.STANDALONE })
-    @Test(groups = { "wso2.esb" })
-    public void testLoadBalanceEndpoint() throws Exception {
-        endPointAdminClient = new EndPointAdminClient(context.getContextUrls().getBackEndUrl(), getSessionCookie());
-
-        cleanupEndpoints();
-        endpointAdditionScenario();
-        endpointStatisticsScenario();
-        endpointDeletionScenario();
     }
 
     @SetEnvironment(executionEnvironments = { ExecutionEnvironment.STANDALONE })
     @Test(groups = { "wso2.esb" }, description = "Sending a Message to a loadbalancing endpoint")
     public void testSendingToLoaBalancingEndpoint()
-            throws IOException, EndpointAdminEndpointAdminException, LoginAuthenticationExceptionException,
-            XMLStreamException {
+            throws IOException {
 
         String response = lbClient.sendLoadBalanceRequest(getProxyServiceURLHttp("loadbalancingEndPoint"), null);
         Assert.assertNotNull(response);
-        Assert.assertTrue(response.toString().contains("Response from server: Server_1"));
+        Assert.assertTrue(response.contains("Response from server: Server_1"));
 
         response = lbClient.sendLoadBalanceRequest(getProxyServiceURLHttp("loadbalancingEndPoint"), null);
         Assert.assertNotNull(response);
-        Assert.assertTrue(response.toString().contains("Response from server: Server_2"));
+        Assert.assertTrue(response.contains("Response from server: Server_2"));
 
         response = lbClient.sendLoadBalanceRequest(getProxyServiceURLHttp("loadbalancingEndPoint"), null);
         Assert.assertNotNull(response);
-        Assert.assertTrue(response.toString().contains("Response from server: Server_3"));
+        Assert.assertTrue(response.contains("Response from server: Server_3"));
```

Comment:
Shall we assert messages?

Reply:
Done in https://github.com/wso2/micro-integrator/pull/574